bucket/say.json

```diff
@@ -13,8 +13,8 @@
     ],
     "bin": "say.ps1",
     "checkver": {
-        "url": "https://api.github.com/repos/lukesampson/psutils/commits",
-        "re": "\"(\\d+)-(\\d+)-(\\d+)",
+        "url": "https://github.com/lukesampson/psutils/commits/master.atom",
```

Comment:
Should not be checked for file directly?

Reply:
This one has assistant script, so checking file may cause error.